### DISCUSSION TITLE
Support values() for QueryDsl

### DISCUSSION THREAD

**Discussion by eloff** [diesel-rs/diesel#3443] [Ideas]
PostgreSQL supports VALUES in the FROM clause for SELECT queries. It acts like a small constant table that you can pass into a query and join with. Would it be possible to support this in diesel? So far it's the most common reason I drop down to raw sql.

See: https://www.postgresql.org/docs/current/queries-values.html

**Comment by weiznich** [comment]
I would consider that as something that's waiting for contributions. For anyone that's interested in implementing this: Please submit a design discussion thread first that outlines the planed API and give a short overview of how you plan to implement this API.
